target_snowflake/db_sync.py

```diff
@@ -679,6 +697,26 @@ def _create_copy_command(pipe_args):
                 self.logger.debug('waiting for snowpipe to transfer data...')
                 time.sleep(next(wait_time))
 
+    def _load_file_copy(self, s3_key, stream, columns_with_trans) -> int:
+        # COPY does insert only
+        inserts = 0
+        with self.open_connection() as connection:
+            with connection.cursor(snowflake.connector.DictCursor) as cur:
+                copy_sql = self.file_format.formatter.create_copy_sql(
+                    table_name=self.table_name(stream, False),
+                    stage_name=self.get_stage_name(stream),
+                    s3_key=s3_key,
+                    file_format_name=self.connection_config['file_format'],
+                    columns=columns_with_trans
```

Comment:
it looks that we need to add changes delivered in the [DATA-3242, ](https://health-union.atlassian.net/browse/DATA-3242)

Reply:
Nice catch, thanks!